macOS/DuckDuckGo/Tab/Navigation/DuckURLSchemeHandler.swift

```diff
@@ -318,7 +318,7 @@ private extension DuckURLSchemeHandler {
         case "svg": return "image/svg+xml"
         case "ico": return "image/x-icon"
         case "riv": return "application/octet-stream"
-        case "json": return "application/json"
+        case "json", "map": return "application/json"
```

Comment:
❓ Out of curiosity, what is this?

Reply:
These are [JavaScript source maps](https://dev.to/inspector/what-are-source-maps-and-how-to-properly-use-them-18h4). They are used for debugging and Shane asked me to add support for serving source map files from duck URLs to simplify his workflow.